src/components/BottomBar/index.js

```diff
@@ -21,59 +20,129 @@ export default class BottomBar extends Component {
   render() {
-    const { classes } = this.props;
-    const { syncBlockTime, syncBlockNum, peerNodeCount, online } = this.props.store.global;
+    const {
+      classes,
+      width,
+      store: {
+        global: { syncBlockTime, syncBlockNum, online },
+        naka: { loggedIn },
+      },
+    } = this.props;
+
     return (
-      <Paper className={classes.bottomBarWrapper}>
-        <NetworkConnection peerNodeCount={peerNodeCount} online={online} />
-        {syncBlockTime && <BlockInfo blockNum={syncBlockNum} blockTime={syncBlockTime} />}
+      <Paper className={classes.paper}>
+        <NetworkConnection width={width} online={online} loggedIn={loggedIn} />
+        {syncBlockTime &&
+          <BlockInfo
+            width={width}
+            blockNum={syncBlockNum}
+            blockTime={syncBlockTime}
+          />
+        }
       </Paper>
     );
   }
 }
 
-const BlockInfo = withStyles(styles)(({ classes, blockNum, blockTime }) => (
-  <Grid container item xs={12} md={7} className={classes.bottomBarBlockInfoWrapper}>
-    <Grid item xs={12} sm={6}>
-      <Typography variant="body2" className={classes.bottomBarTxt}>
-        <span className={classes.bottomBarBlockNum}><FormattedMessage id="bottomBar.blockNum" defaultMessage="Current Block Number" />{`: ${blockNum}`}</span>
-      </Typography>
-    </Grid>
-    <Grid item xs={12} sm={6}>
-      <Typography variant="body2" className={classes.bottomBarTxt}>
-        <span className={classes.bottomBarBlockTime}><FormattedMessage id="bottomBar.blockTime" defaultMessage="Current Block Time" />: {blockTime ? moment.unix(blockTime).format('LLL') : ''}</span>
-      </Typography>
-    </Grid>
-  </Grid>
-));
-
-const NetworkConnection = withStyles(styles)(({ classes, peerNodeCount, online }) => (
-  <Grid container item xs={12} md={6} className={classes.bottomBarNetworkWrapper}>
-    {online ? (
+const NetworkConnection = withStyles(styles)(({ classes, width, online, loggedIn }) => (
+  <Grid container item xs={12} md={6}>
+    {width !== 'xs' && (
+      <Fragment>
+        <div className={classes.statusContainer}>
+          {online
+            ? <CheckCircle className={cx(classes.statusIcon, 'online')} />
+            : <RemoveCircle className={cx(classes.statusIcon, 'offline')} />
+          }
+          <Typography variant="body2" className={cx(classes.bottomBarTxt, 'network')}>
+            <FormattedMessage id="bottomBar.network" defaultMessage="Network" />
+          </Typography>
+        </div>
+        <div className={classes.statusContainer}>
+          {loggedIn
+            ? <CheckCircle className={cx(classes.statusIcon, 'online')} />
+            : <RemoveCircle className={cx(classes.statusIcon, 'offline')} />
+          }
+          <Typography variant="body2" className={classes.bottomBarTxt}>
+            <FormattedMessage id="bottomBar.wallet" defaultMessage="Wallet" />
+          </Typography>
+        </div>
+      </Fragment>
+    )}
+    {width === 'xs' && (
```

Comment:
Does material ui have grid system can directly apply responsive page instead of having the code duplicate?

Reply:
nope